plover/gui/add_translation.py

```diff
@@ -202,6 +205,20 @@ def on_move(self, event):
         self.config.set_translation_frame_y(pos[1])
         event.Skip()
 
+    def applyTransparency(self, transparency):
+        """
+        Given a transparency in range 0 through 100, makes the window that
+        percent transparent. Handles conversion to range 0 through 0xFF.
+        """
+        if not self.CanSetTransparent():
```

Comment:
We should probably either disable or not show the transparency UI in Config when we `not CanSetTransparent`.